tests/plotting/test_widgets.py

```diff
@@ -346,24 +379,157 @@ def test_add_volume_clip_plane(uniform):
+    width, height = pl.window_size
+    pl.iren._mouse_move(width // 2, height // 2)
+    assert widget._selected_actor in widget._arrows + widget._circles
+    assert widget._selected_actor.prop.color == pv.Color(DARK_YELLOW)
+
+    # ensure that the actor gets deselected
+    pl.iren._mouse_move(0, 0)
+    assert not widget._selected_actor
+
+    # test X axis translation
+    pl.iren._mouse_left_button_press(width // 2 - 1, height // 2 - 1)
+    assert widget._selected_actor is widget._arrows[0]
+    assert widget._pressing_down
+    pl.iren._mouse_move(width, height // 2)
+    assert actor.user_matrix[0, 3] < 0
+    pl.iren._mouse_left_button_release(width, height // 2)
+    assert actor.user_matrix[0, 3] < 0
+
+    # test callback called
+    assert len(calls) == 1 and calls[0].shape == (4, 4)
+
+    # test Y axis translation
+    pl.iren._mouse_left_button_press(width // 2 + 1, height // 2 - 1)
+    assert widget._selected_actor is widget._arrows[1]
+    assert widget._pressing_down
+    pl.iren._mouse_move(width, height // 2)
+    assert actor.user_matrix[1, 3] < 0
+    pl.iren._mouse_left_button_release()
+    assert actor.user_matrix[1, 3] < 0
+
+    # test Z axis translation
+    pl.iren._mouse_left_button_press(width // 2, height // 2 + 5)
+    assert widget._selected_actor is widget._arrows[2]
+    assert widget._pressing_down
+    pl.iren._mouse_move(width // 2, 0)
+    assert actor.user_matrix[3, 3] > 0
+    pl.iren._mouse_left_button_release()
+    assert actor.user_matrix[3, 3] > 0
+
+    # test X axis rotation
+    pl.iren._mouse_left_button_press(width // 2 + 30, height // 2)
+    assert widget._selected_actor is widget._circles[0]
+    assert widget._pressing_down
+    pl.iren._mouse_move(width // 2 + 30, height // 2 + 1)
+    x_r, y_r, z_r = r_mat_to_euler_angles(actor.user_matrix)
+    assert x_r > 0 and np.allclose([y_r, z_r], 0)
+    pl.iren._mouse_left_button_release()
+    assert not widget._pressing_down
+    widget._reset()
+    assert np.allclose(widget._cached_matrix, np.eye(4))
+
+    # test Y axis rotation
+    pl.iren._mouse_left_button_press(width // 2 - 30, height // 2)
+    assert widget._selected_actor is widget._circles[1]
+    assert widget._pressing_down
+    pl.iren._mouse_move(width // 2 - 30, height // 2 - 1)
+    x_r, y_r, z_r = r_mat_to_euler_angles(actor.user_matrix)
+    assert y_r > 0 and np.allclose([x_r, z_r], 0)
+    pl.iren._mouse_left_button_release()
+    assert not widget._pressing_down
+    widget._reset()
+
+    # test Z axis rotation
+    pl.iren._mouse_left_button_press(width // 2, height // 2 - 28)
+    assert widget._selected_actor is widget._circles[2]
+    assert widget._pressing_down
+    pl.iren._mouse_move(width // 2 - 1, height // 2 + 30)
+    x_r, y_r, z_r = r_mat_to_euler_angles(actor.user_matrix)
+    assert z_r > 0 and np.allclose([x_r, y_r], 0)
+    pl.iren._mouse_left_button_release()
+    assert not widget._pressing_down
+    widget._reset()
```

Comment:
Let's increase coverage!
```suggestion
    widget._reset()

    # test origin
    origin = np.random.random(3)
    widget.origin = origin
    assert np.allclose(widget.origin, origin)

    # test disable
    widget.disable()
```